### PR TITLE
AG-8002 - Adjust gallery example year strategy.

### PR DESCRIPTION
**Pull request by alantreadway**
https://ag-grid.atlassian.net/browse/AG-8002

- Undoes the dynamic year in gallery examples.
- Removes years from gallery examples so they appear less dated.
- Moved source subtitles into footnotes.

### CODE REVIEW

**Review by lsjroberts — APPROVED** [review]
There's a few places that use the year still, any alternative to that or not worrying about it?

**Comment by alantreadway** [comment]
> There's a few places that use the year still, any alternative to that or not worrying about it?

For older data-sets (e.g. 1987, 2010) we decided it was probably OK, so the update is mainly focusing on data-sets in recent years - LMK if we missing any spots though?

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-overview/examples/simple-line/data.ts [hunk 1]
```diff
@@ -1,58 +1,57 @@
...
+        { date: new Date(2019, 9, 28), petrol: 125.77, diesel: 130.6 },
+        { date: new Date(2019, 10, 4), petrol: 125.56, diesel: 130.38 },
+        { date: new Date(2019, 10, 11), petrol: 125.59, diesel: 130.42 },
+        { date: new Date(2019, 10, 18), petrol: 125.58, diesel: 130.35 },
+        { date: new Date(2019, 10, 25), petrol: 125.32, diesel: 130.08 },
+        { date: new Date(2019, 11, 2), petrol: 124.81, diesel: 129.79 },
+        { date: new Date(2019, 11, 9), petrol: 124.75, diesel: 129.79 },
+        { date: new Date(2019, 11, 16), petrol: 124.33, diesel: 129.56 },
+        { date: new Date(2019, 11, 23), petrol: 124.16, diesel: 129.81 },
+        { date: new Date(2019, 11, 30), petrol: 124.96, diesel: 130.54 },
```
Comment (lsjroberts) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-overview/examples/simple-line/data.ts:55]: 2019 ^

**Comment by alantreadway** [comment]
Going for merge since this is quite a broad PR.

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-overview/examples/stacked-area/data.ts [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+        { date: new Date(Date.UTC(2019, 8, 1)), "Science Museum": 196336, "National Media Museum": 29462, "National Railway Museum": 55506, "Locomotion": 13825, "Museum of Science and Industry, Manchester": 39673, "National Coal Mining Museum for England": 8216, },
+        { date: new Date(Date.UTC(2019, 9, 1)), "Science Museum": 312781, "National Media Museum": 42880, "National Railway Museum": 73100, "Locomotion": 17296, "Museum of Science and Industry, Manchester": 51834, "National Coal Mining Museum for England": 8444, },
+        { date: new Date(Date.UTC(2019, 10, 1)), "Science Museum": 254938, "National Media Museum": 29745, "National Railway Museum": 51243, "Locomotion": 13826, "Museum of Science and Industry, Manchester": 35991, "National Coal Mining Museum for England": 5903, },
+        { date: new Date(Date.UTC(2019, 11, 1)), "Science Museum": 237880, "National Media Museum": 26347, "National Railway Museum": 49346, "Locomotion": 7492, "Museum of Science and Industry, Manchester": 32697, "National Coal Mining Museum for England": 6409, },
```
Comment (lsjroberts) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-overview/examples/stacked-area/data.ts:15]: 2019 ^

**Pull request merged**